templates/default/dockerImage.sh.erb

```diff
@@ -46,6 +46,27 @@ elif [ "$1" == "gc" ] ; then
     docker exec registry bin/registry garbage-collect /etc/docker/registry/config.yml -m
 	  docker container restart registry
 
+elif [ "$1" == "delete-acr" ] ; then
```

Comment:
This should be under the else of this condition when `managed_docker_registry` is true 
https://github.com/logicalclocks/hopsworks-chef/pull/570/files#diff-944e67fb3fc477da8e732aeac8d41b01b35b98af540418d278610340363d1454R30

Reply:
ahhh I read if managed=true. fixed in last commit

Reply:
The block was correct you just need to add `<% else -%>` before it

Reply:
Ah sorry, I am not awake today.. fixed now

Reply:
no worries